content/blog/2018-02-10-drake.md

```diff
@@ -0,0 +1,117 @@
+---
+slug: "drake"
+title: "The prequel to the drake R package"
+package_version: 5.0.0
+authors:
+  - name: Will Landau
+    url: https://github.com/wlandau
+date: 2018-02-10
+categories: blog
+topicid: 1033
+tags:
+- r
+- community
+- software
+- review
+- onboarding
+- package
+- drake
+- reproducibility
+- high-performance-computing
+- pipeline
+- workflow
+output:
+  html_document:
+    keep_md: yes
+---
+
+The [drake](https://github.com/ropensci/drake) package is a [pipeline](https://github.com/pditommaso/awesome-pipeline) toolkit. It manages data science workflows, [saves time](https://ropensci.github.io/drake/#what-gets-done-stays-done-), and [ensures a kind of reproducibility](https://ropensci.github.io/drake/#reproducibility-with-confidence) that the R community seldom thinks about. I am trying to change the conversation around reproducible research, and I am using [drake](https://github.com/ropensci/drake) as a flagship. However, I originally created [drake](https://github.com/ropensci/drake) for totally different reasons. This post is the prequel to [drake](https://github.com/ropensci/drake)'s inception. There was struggle, and [drake](https://github.com/ropensci/drake) was the answer.
+
+# Dissertation frustration
+
+<center><img src="/img/blog-images/2018-02-10-drake/uphill.png" style = "width: 300px" alt="Sisyphus"></center>
```

Comment:
Please credit images in text below image. They seem to come from here: https://sites.google.com/site/sisyphusa/

See example img credit https://ropensci.org/blog/2017/11/17/unconf-sixtips/

`<img src="/img/blog-images/2017-11-17-unconf-sixtips/ropensci-unconf17-community-nistara-randawa.jpg" alt="Attendees at the May 2017 rOpenSci unconference. Photo credit: Nistara Randhawa">
_Attendees at the May 2017 rOpenSci unconference. Photo credit: Nistara Randhawa_`